core-httpclient-impl/src/main/java/com/optimizely/ab/OptimizelyFactory.java

```diff
@@ -369,11 +388,20 @@ public static Optimizely newDefaultInstance(ProjectConfigManager configManager,
             .withApiManager(odpApiManager != null ? odpApiManager : new DefaultODPApiManager())
             .build();
 
+        // If no cmabService provided, create default one
+        if (cmabService == null) {
+            DefaultCmabClient defaultCmabClient = new DefaultCmabClient(CmabClientConfig.withDefaultRetry());
```

Comment:
Can we change the DefaultCmabClient to use defaultRetry by default, so we can use simpler 
 ```suggestion
            DefaultCmabClient defaultCmabClient = new DefaultCmabClient();
```